src/shadowbox/server/main.ts

```diff
@@ -34,7 +34,7 @@ import * as server_config from './server_config';
 import {OutlineSharedMetricsPublisher, PrometheusUsageMetrics, RestMetricsCollectorClient, SharedMetricsPublisher} from './shared_metrics';
 
 const DEFAULT_STATE_DIR = '/root/shadowbox/persisted-state';
-const MMDB_LOCATION = '/var/lib/libmaxminddb/GeoLite2-Country.mmdb';
+const MMDB_LOCATION = '/var/lib/libmaxminddb/dbip-country-lite.mmdb';
```

Comment:
We should move to the persisted-state: we are losing the updates when we restart the server

Reply:
I'm not sure what you mean by this, but this script is run at image build time so I don't think restarting the server would have any effect on this being here.  I could be wrong?